src/table/utils/handle-accessibility.ts

```diff
@@ -0,0 +1,162 @@
+import { BaseSyntheticEvent } from 'react';
+import { stardust } from '@nebula.js/stardust';
+import { AnnounceFn, TableCell, TotalsPosition } from '../../types';
+
+interface ICellFocusProps {
+  focusType: string;
+  cell: HTMLTableCellElement | undefined;
+}
+interface IHandleResetFocusProps {
```

Comment:
It would be very good to put these interfaces to the types.ts file to shorten the line of code of this file.

Reply:
And we do not add 'I' at the beginning of an interface.

Reply:
Maybe we should though. Probably best to remove `I` now for consistency, but we should look into best practice later